pyspades/master.py

```diff
@@ -96,13 +97,16 @@ def on_disconnect(self):
             self.defer = None
         callback = self.disconnect_callback
         if callback is not None:
-            callback()
+            callback(self)
         self.disconnect_callback = None
 
 
 def get_master_connection(protocol):
-    defer = Deferred()
-    connection = protocol.connect(MasterConnection, HOST, PORT, MASTER_VERSION)
-    connection.server_protocol = protocol
-    connection.defer = defer
-    return defer
+    defers = []
+    for host in HOSTS:
+        defer = Deferred()
+        connection = protocol.connect(MasterConnection, host, PORT, MASTER_VERSION)
```

Comment:
We're running into a problem here. When creating the `enet.Host`, we need to specify the amount of Enet connections we are connecting to. We are currently allocating 34 of those, and we need to add more of those for additional masters.